### ISSUE TITLE
data.snowflake_accounts doesn't work

### ISSUE THREAD

**Issue by bogdan-cioata** [Snowflake-Labs/terraform-provider-snowflake#2158]
**Provider Version**

0.68.2

**Terraform Version**

1.6.1

**Describe the bug**

[data.snowflake_accounts](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/data-sources/accounts) returns an unexpected error when called using `ORGADMIN` provider: 
```
Error: sql: Scan error on column index 6, name "comment": converting NULL to string is unsupported
│ 
│   with data.snowflake_accounts.domain,
│   on main.tf line 47, in data "snowflake_accounts" "domain":
│   47: data "snowflake_accounts" "domain" {
```
Calling it using `ACCOUNTADMIN` provider succeeds, but returns `null` values.

**Expected behavior**

Expected to return the list of accounts with the given attributes, as described in the [docs](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/data-sources/accounts#nested-schema-for-accounts).

**Code samples and commands**
```
data "snowflake_accounts" "domain" {
  provider = snowflake.orgadmin
}
```

**Additional context**

NA

**Comment by sfc-gh-asawicki** [comment]
Hey @bogdan-cioata.

It was fixed in #2148 and has been available since v0.75.0. Because there were problems between versions 0.73.0 and 0.76.0, please bump it to at least v0.76.0 and confirm that the fix worked.

**Comment by bogdan-cioata** [comment]
Hey @sfc-gh-asawicki,

Works fine now, thanks :)
